tests/generation/test_logits_process.py

```diff
@@ -304,6 +305,53 @@ def test_top_p_dist_warper(self):
         # first batch should keep three tokens, second batch would keep only 1, but due to `min_tokens_to_keep=2` keeps 2.
         self.assertListEqual((filtered_dist != 0.0).to(torch.long).sum(dim=-1).tolist(), [3, 2])
 
+    def test_min_p_dist_warper(self):
+        input_ids = None
+        vocab_size = 10
+        batch_size = 2
+
+        # create distribution and take log (inverse to Softmax as taken in MinPLogitsWarper)
+        dist = torch.log(
+            torch.tensor(
+                [
+                    [0.9, 0.047, 0.0274, 0.0274],  # the two first tokens should be kept (0.9*0.05=0.045)
```

Comment:
Could we switch the order here e.g. `[0.9, 0.0274, 0.047, 0.0274]` just to make sure the selection using sorted indices and masking is working as expected i.e. it doesn't just take the first n tokens, regardless of their value